requirements/requirements.txt

```diff
@@ -1,6 +1,6 @@
 aiohttp>=3.5.2,<4.0
 aiohttp_cors>=0.7,<2.0
-aiokafka==0.7.0
```

Comment:
wouldn't be better to put it as 
```
aiokafka>=0.7.1,<0.8.0
```
?

Reply:
Could you update the PR with the requested changes? I can approve it once the updated version is pushed and do another release